pliers/support/download_embedding.py

```diff
@@ -0,0 +1,165 @@
+import nltk
+import os
+import requests
+import sys
+import tarfile
+import logging
+
+import gensim
+from gensim.scripts.glove2word2vec import glove2word2vec,get_glove_info
+import argparse
+
+_aws_bucket_path = 'https://s3.amazonaws.com/mlt-word-embeddings/'
```

Comment:
Is this bucket completely public? Does it require set of AWS credentials?

Reply:
Hi qmac, the bucket belongs to the machine learning team at NIMH, and is public. We use it to share embeddings and models with other researchers, since the files are large. Debanjan and I thought it would make sense to develop the code with a single source for all the embeddings and models, and since we already had them all in one place... We will be happy to contemplate other possibilities in the longer term.

Reply:
Sounds good to me, thanks!

Reply:
Yeah I worry a little bit about guaranteeing that these locations will always be available, but I suppose that's the case for any pre-trained model that needs to be downloaded.